src/ostree/ot-builtin-commit.c

```diff
@@ -597,6 +600,19 @@ ostree_builtin_commit (int argc, char **argv, OstreeCommandInvocation *invocatio
     flags |= OSTREE_REPO_COMMIT_MODIFIER_FLAGS_SKIP_XATTRS;
   if (opt_consume)
     flags |= OSTREE_REPO_COMMIT_MODIFIER_FLAGS_CONSUME;
+  switch (opt_selinux_labeling_epoch)
+    {
+    case 0:
+      break;
+    case 1:
+      flags |= OSTREE_REPO_COMMIT_MODIFIER_FLAGS_SELINUX_LABEL_V1;
```

Comment:
Right the background for this is that today there's `ostree admin init-fs --modern` and I actually wanted to change the defaults there to do something else, and then it gets weird because it'd be need to be called like `--really-modern` or something :smile: 

You're probably right we wouldn't change things again (and if we *did* we'd probably arguably want to change the policy defaults).

But still, no harm done in making this a bit more future proof right?

Reply:
> But still, no harm done in making this a bit more future proof right?

Just trying to keep out unnecessary complexity (YAGNI and all that). :)
But not strongly against.